fdbserver/storageserver.actor.cpp

```diff
@@ -13803,6 +13803,35 @@ ACTOR Future<Void> storageInterfaceRegistration(StorageServer* self,
 	return Void();
 }
 
+ACTOR Future<Void> rocksdbLogCleaner(std::string folder) {
+	std::replace(folder.begin(), folder.end(), '/', '_');
+	if (!folder.empty() && folder[0] == '_') {
+		folder.erase(0, 1);
+	}
+	try {
+		loop {
+			wait(delayJittered(SERVER_KNOBS->STORAGE_ROCKSDB_LOG_CLEAN_UP_DELAY));
+			TraceEvent("CleanUpRocksDBLogs").detail("LogPrefix", folder);
+			auto logFiles = platform::listFiles(SERVER_KNOBS->LOG_DIRECTORY);
```

Comment:
Looks like it deletes all files under this directory. If yes, probably we need to change the function name.

Reply:
The trace logs have a different naming conversion. They will not be deleted.